src/mixins/raster-layer-poly-mixin.js

```diff
@@ -495,14 +427,9 @@ export default function rasterLayerPolyMixin(_layer) {
         ),
         default: adjustOpacity(
           polyDefaultScaleColor,
-          state.encoding.color.hasOwnProperty("showOther") &&
-            state.encoding.color.showOther === false
-            ? 0
-            : state.encoding.color.opacity
-            ? state.encoding.color.opacity
-            : 0.65
+          hasShowOther ? 0 : state.encoding.color.opacity || 0.65
```

Comment:
the `0.65` magic number should be put in a global default variable for opacity. Obviously that predates you, so no need to do anything about it here. Just noting.

Reply:
Easy to update, i'll make the change along with the extra validation changes